devel/libdynd/Portfile

```diff
@@ -23,6 +23,9 @@ checksums           rmd160  58a444d46d348fddc6f4cf04669d301dbb47981e \
 
 patchfiles-append   patch-CMakeLists.txt.diff
 
+# https://trac.macports.org/ticket/59745
+patchfiles-append   patch-src_dynd_type.cpp.diff
+
```

Comment:
Revision is not incremented because the code in the patch behaves identically to as before.